docs/package_integration/uds-package-practices.md

```diff
@@ -1,15 +1,26 @@
-# UDS Package Practices
+# UDS Package Integration Guide
 
-This document describes the practices that a UDS package **must**, **should** and **may** follow to be considered officially supported as a UDS package and to be able to display the "Made for UDS" badge on its GitHub repository README.md file :
+## Introduction
 
-[<img alt="Made for UDS" src="./made-for-uds.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core)
+This guide is intended for developers integrating applications with UDS (Unicorn Delivery Service). This document describes the practices that a UDS package _**must**_, _**should**_ and _**may**_ follow to be considered officially supported as a UDS package.
```

Comment:
do we need these edits here? isn't the other file the guide?

Reply:
Good catch, My copy/paste fell short. will update